tests/test_wsireader.py

```diff
@@ -1,18 +1,24 @@
 """Tests for reading whole-slide images."""
 
-from tiatoolbox.wsicore import wsireader
-from tiatoolbox import utils
-from tiatoolbox import cli
-from tiatoolbox.utils.exceptions import FileNotSupported
-
-import pytest
-from pytest import approx
 import pathlib
-import numpy as np
+
+# import sys
```

Comment:
remove commented import lines.

Reply:
Fixed in https://github.com/TIA-Lab/tiatoolbox/pull/123/commits/ac77bdc8bbf73150178286e8e68af148d9372cc6